lms/static/scripts/frontend_apps/components/dashboard/test/DashboardActivityFilters-test.js

```diff
@@ -270,6 +259,52 @@ describe('DashboardActivityFilters', () => {
     });
   });
 
+  describe('clear filters', () => {
+    [
+      // Callback provided, but no items selected
+      {
+        props: { onSelectionReset: sinon.stub() },
+        shouldRenderClearButton: false,
+      },
+      // Callback not provided
+      {
+        props: {
+          selectedAssignmentIds: [...assignments],
+          selectedStudentIds: [...studentsWithName],
+        },
+        shouldRenderClearButton: false,
+      },
+      // Callback provided and items selected
+      {
+        props: {
+          onSelectionReset: sinon.stub(),
+          selectedCourseIds: [...courses],
+        },
+        shouldRenderClearButton: true,
+      },
+    ].forEach(({ props, shouldRenderClearButton }) => {
+      it('shows clear button if onSelectionReset callback was provided and some items are selected', () => {
```

Comment:
```suggestion
      it('shows clear button if `onSelectionReset` callback was provided and some items are selected', () => {
```

In this context I think it is helpful to highlight the identifier. Same for the test below.